Automattic/woocommerce-payments
# Introduce the `wcpay_terminal_payment_completed_order_status` filter

#### Changes proposed in this Pull Request
This pull request introduces the filter `wcpay_terminal_payment_completed_order_status`, which allows for the alteration of the order status from 'completed' to 'processing' following a successful terminal charge.

More context: pdfdoF-3XW-p2

#### Testing instructions
- Add a new plugin to your `mu-plugins` folder with the following code: `add_filter( 'wcpay_terminal_payment_completed_order_status', function() { return 'processing'; } )`.
- Add a product to your cart.
- Finish the order by paying with a card reader or a simulated In-Person payment order.
- Verify that the order status is `processing` instead of `completed`.

*

-------------------


**Post merge**
